### DISCUSSION TITLE
[suggestion] BASIC-like drawing

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#291] [Ideas]
SDL2 is too hard for me to wrap my mind around. BASIC-like drawing is much simpler and powerful enough for me I think. What do you think about adding a library to provide BASIC-like drawing to Objeck?

You could use SmallBASIC as a reference:

https://github.com/smallbasic/SmallBASIC

BASIC-like drawing will make drawing on Objeck much easier and more fun.

**Comment by objeck** [comment]
Can you provide a link to the 2D drawing APIs?

**Reply by ghost** [reply]
https://smallbasic.github.io/pages/graphics.html

**Comment by ghost** [comment]
I think I will try to learn SDL2 instead.
